barretenberg/cpp/src/barretenberg/crypto/schnorr/c_bind.hpp

```diff
@@ -29,11 +29,11 @@ WASM_EXPORT void schnorr_multisig_construct_signature_round_1(
 
 WASM_EXPORT void schnorr_multisig_construct_signature_round_2(
     uint8_t const* message,
-    fq::in_buf private_key,
```

Comment:
mind tl;dr'ing why we're switching field types?

Reply:
This was an oopsie. Fixed. It was actually another api call that was meant to become Fq.
Typescript didnt raise error due to Fq and Fr being structurally the same, although this PR changes that as I optimised Fr slightly.